frontend/src/theme/components/Tabs.ts

```diff
@@ -0,0 +1,116 @@
+import {
+  ChakraTheme,
+  ComponentMultiStyleConfig,
+  SystemStyleObjectRecord,
+  ThemingPropsThunk,
+} from '@chakra-ui/react'
+import { merge } from 'lodash'
+
+const variantLine: ThemingPropsThunk<SystemStyleObjectRecord, ChakraTheme> = (
+  props,
+) => {
+  const { orientation } = props
+  const isVertical = orientation === 'vertical'
+  const marginProp = isVertical ? 'marginStart' : 'marginBottom'
+  return {
+    root: {
+      w: 'auto',
+      p: '24px',
+      mx: '-16px', // ignore parent padding and extend full width of screen
```

Comment:
Is this because we are overriding `line` styles? Will probably not be needed with our own variant

Reply:
Realised the padding is from storybook so this line is not needed